src/schema.py

```diff
@@ -20,7 +20,6 @@ class JobSchema(BaseModel):
     files_in_progress: int
     total_files: int
     files_errored: int
-    iterator: Optional[str]
```

Comment:
Unused since long time ago (iterators are not a global property of job since distributed workers are allowed == long time ago).